open-api/README.md

```diff
@@ -40,3 +40,66 @@ make generate
 ```
 
 The generated code is not being used in the project, but helps to see what the changes in the open-API definition are in the generated code.
+
+# REST Compatibility Kit (RCK)
+
+The REST Compatibility Kit (RCK) is a Technology Compatibility Kit (TCK) implementation for the 
+Iceberg REST Specification.  This includes a series of tests based on the Java reference
+implementation of the REST Catalog that can be executed against any REST server that implements the
+spec.
```

Comment:
maybe nit-picky, but I do believe RCK (TCK) ought to be based on the _spec_, not on the reference implementation. The RCK should _pass_ on the reference impl.. but the foundation for it is the spec. Technically, the reference impl. may be incorrect with the respect to the spec (as any piece of software), so the test ought to validate that as well as catalog implementations from other projects.

Reply:
I agree the intent is the RCK is based on the Spec and I don't feel this is saying otherwise.  The purpose of the reference implementation is to faithfully implement the spec and the tests should validate that behavior.  We're using the reference implementation tests as part of this because the "should" achieve the same objective.  Creating a separate set of tests for the RCK would either be duplicative or imply that the reference implementation doesn't adhere to the spec (neither of which is good).